pimd/pim_cmd.c

```diff
@@ -3415,15 +3415,24 @@ static void igmp_show_groups(struct pim_instance *pim, struct vty *vty, bool uj)
 	time_t now;
 	json_object *json = NULL;
 	json_object *json_iface = NULL;
-	json_object *json_row = NULL;
+	json_object *json_group = NULL;
```

Comment:
is this somewhere freed later or I just don't see that?

Reply:
json_object_free(json); at line 3528